cmd/snap/cmd_snap_op.go

```diff
@@ -118,10 +118,11 @@ func (x *cmdInstall) Execute([]string) error {
 	var err error
 
 	cli := Client()
-	if strings.Contains(x.Positional.Snap, "/") {
-		uuid, err = cli.InstallSnapFile(x.Positional.Snap)
+	name := x.Positional.Snap
+	if strings.Contains(name, "/") || strings.HasSuffix(name, ".snap") || strings.Contains(name, ".snap.") {
```

Comment:
@niemeyer note chrome adds ` (n)` to the filename, e.g. `http.chipaca_4.66920_all (1).snap`